tests/serverpod_test_server/lib/src/protocol/types.yaml

```diff
@@ -1,9 +1,19 @@
 class: Types
```

Comment:
Please make sure that no changes are made to these files

Reply:
ok i think that it will help when testing

Reply:
i replaced all data in this test folder as before in https://github.com/serverpod/serverpod/pull/194